src/test/kotlin/com/atlassian/performance/tools/report/jstat/JstatConverterTest.kt

```diff
@@ -25,4 +25,16 @@ class JstatConverterTest {
 
         assertThat(actualCsv, equalTo(expectedCsv))
     }
+
+    @Test
+    fun shouldCreateFreshCsvOnEachRun() {
```

Comment:
It keeps failing for me, both on 1a8f0ef229160aa04c12b593350aa9e3b4b4d55f and 18440112d4b082be2534a0373aa241498fb74e29.
[test.zip](https://github.com/atlassian/report/files/3093749/test.zip)

It's green when I change it to this:
```kotlin
    @Test
    fun shouldCreateFreshCsvOnEachRun() {
        val logFile = folder.newFile("jstat.log")
        this::class.java.getResourceAsStream("./jstat.log").copyTo(logFile.outputStream())

        val firstCsvResult = jstatConverter.convertToCsv(logFile.toPath()).toFile()
        val secondCsvResult = jstatConverter.convertToCsv(logFile.toPath()).toFile()

        Assertions.assertThat(firstCsvResult).hasSameContentAs(secondCsvResult)
    }
```
Does it work for you locally? I'm using Windows.

Reply:
Yes. Works on mac.

Reply:
Maybe it was never working on windows.

Reply:
Hmm. IMHO we should make the test coverage multi-platform, because our lib (`src/main`) is supposed to be multiplatform. E.g. we shouldn't force all users to get Macs (Atlassians, app vendors, customers). Also, Java itself is multiplatform, so we're really close to achieving that.

PS. welp, my fix obviously is wrong (green on both commits), because I'm reading from the same file twice *after* both conversions.

I debugged the test and in `JstatConverter`, the `it.delete()` returns `false`.

Reply:
Agree, but we don't have CI to cover windows, and I don't have windows to test it. Making it back windows compatible is out of scope for his PR.

Reply:
It's a new test, so it is the perfect time to make it good.